tests/test_layers/test_orthogonal.py

```diff
@@ -10,10 +12,15 @@
 
 list_prohibited_matrices_dim0 = list_matrices[:-1]
 
-def test_constructor():
-    orth = OrthogonalBlock(1) 
-    orth = OrthogonalBlock(0) 
-    orth = OrthogonalBlock()
+@pytest.mark.parametrize("dim", [-1, 0, 1])
+def test_constructor(dim):
+    OrthogonalBlock(dim) 
```

Comment:
Check also for `None` argument

Reply:
What should be the right behaviour with ``None``?

Reply:
I meant the default parameters.
https://github.com/mathLab/PINA/blob/8ec5fd6e210e5d502366e9f85c5cfee1c8d8e0f5/pina/model/layers/orthogonal.py#L14

The case when you don't pass any argument is never checked.

Reply:
Done! Thanks